### PR TITLE
feat: dumpconfig also dump disabled plugin

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: lib/egg.js [hunk 1]
```diff
@@ -372,7 +372,7 @@ class EggApplication extends EggCore {
       ignoreList = [];
     }
 
-    const json = extend(true, {}, { config: this.config, plugins: this.plugins });
+    const json = extend(true, {}, { config: this.config, plugins: this.loader.allPlugins });
```
Comment (atian25) [inline lib/egg.js:375]: 比起之前的变更：
- 多了 disable 的插件
- 没有排序

**Comment by killagu** [comment]
CI 需要修复一下

**Comment by atian25** [comment]
> CI 需要修复一下

修了，换了个 fixtures，可以再看下

**Review by killagu — APPROVED** [review]
LGTM

**Pull request merged**